api/app.py

```diff
@@ -23,9 +67,13 @@ def ping():
 
 
 @app.on_event("startup")
-def Initialize():
-    global con
-    con = ConnectDuckDB()
+async def Initialize():
```

Comment:
Python function names are generally all lowercase with words separated by an underscore.  I think you follow that elsewhere as well.  Let's stick to that.

Classes are generally camelcase with the first letter of each word as a capital letter.

Reply:
yes, that's right, I've made the changes, thanks for the suggestion!